### ISSUE TITLE
PHP 7.1 Suport

### ISSUE THREAD

**Issue by larsroettig** [Inchoo/Inchoo_PHP7#141]
Hi you have a plan for merge DEV in the Master and Support php 7.1 ?

best regards,

Lars

**Comment by piotrekkaminski** [comment]
@larsroettig we are working together with Inchoo to have a patch for 1.9.2.x+/1.14.3.x+ based on the dev branch for 7.2. Expected before EOL of 5.6.

**Comment by hostep** [comment]
@piotrekkaminski: no patches planned for Magento 1.7/1.8/1.9.0/1.9.1? 😞 
(I can understand you don't want to put too much time into this, but unfortunately we still have to support some of those older versions because merchants don't want to pay for upgrades)

**Comment by piotrekkaminski** [comment]
@hostep unfortunately no - those versions were not tested with PHP 5.6 even.

**Comment by hostep** [comment]
Ok, yes indeed unfortunately.

This is only FYI: but we are running several Magento 1.9.0.x and 1.9.1.x projects on PHP 7.0 with the help of https://github.com/EliasKotlyar/MagentoPHP56 (with some tweaks) & this very own Inchoo_PHP7 plugin.

**Comment by erikhansen** [comment]
Any update on when support for PHP 7.1 will be released? PHP 5.6 and PHP 7.0 [are reaching EOL](http://php.net/supported-versions.php) on December 31st and with only four months until then, we'd like to start moving all our clients to PHP 7.1 in the next month.

**Comment by erikhansen** [comment]
@piotrekkaminski Do you have an update on this?

**Comment by piotrekkaminski** [comment]
Yes - the patches are in beta testing and will be published soon.

**Comment by ivanweiler** [comment]
Already mentioned in another issue, Magento official PHP 7.2 patches are released. Use official patches, this extension is not needed anymore.

Patches:
https://magento.com/tech-resources/download#download2240
More info:
https://inchoo.net/magento/magento-1-official-php-7-2-patches/
